core/src/main/java/hudson/cli/LoginCommand.java

```diff
@@ -45,6 +46,8 @@ protected int run() throws Exception {
         ClientAuthenticationCache store = new ClientAuthenticationCache(checkChannel());
         store.set(a);
 
+        SecurityListener.fireLoggedIn(a.getName());
```

Comment:
Makes [this Javadoc](https://github.com/jenkinsci/jenkins/blob/bcbd5ba56ad7bbed00977b2d733f60019c6af04c/core/src/main/java/jenkins/security/SecurityListener.java#L63) inaccurate.